client/my-sites/plan-features/index.jsx

```diff
@@ -413,9 +419,24 @@ class PlanFeatures extends Component {
 				planName
 			} = properties;
 
-			const featureKeys = Object.keys( features ),
+			let abTestVariantFeatures = features;
+
+			const premiumSquaredShowMarketingCopy = abtest( 'premiumSquaredPlansWording' ) === 'withMarketingCopy';
```

Comment:
We generally try avoiding calling abtest until we're sure we're targeting our target users. This avoids a test start event when requirements haven't been met (eg needs permissions, or some other segment). It doesn't matter much in this case, but can be important to reduce noise in other tests.

 We're also calling this multiple times in the map function which can be avoided.